### PR TITLE
README updates. Adds note about beta/master

### PR DESCRIPTION
**Pull request by AngellusMortis**
* Removes note about EOL of non-UniFiOS devices
* Adds section about supported versions UniFi Protect
* Various spelling/markdown formating fixes

I also did not make/use a fork since I have not ever edited this repo. If we want to still use/enforce forks, let me know.

### CODE REVIEW

**Comment by briis** [comment]
No problem editing directly. I actually also started on this and saved it in README_0.10.0.md Here I also changed the pictures etc. While you are at it, could you look at that and merge in these changes?

**Comment by AngellusMortis** [comment]
Sure thing. Let me merge those.

File: README.md [hunk 1]
```diff
@@ -2,68 +2,90 @@
...
+
+**Stable versions of this integration require the latest stable version of UniFi Protect to run.**
+
+**Beta versions / `master` branch of this integration require the latest beta version of UniFi Protect to run (or the latest stable if there is no beta)**
+
+We try our best to avoid breaking changes so you may need to use older versions of UniFi Protect with newer versions of the integration. Just keep in mind, we may not be able to support you if you do.
+
+## Docs for Old Versions
+
+If you are not using the latest beta of the integration, you can view old versions of this README at any time in Github at `https://github.com/briis/unifiprotect/tree/{VERSION}`. Example, docs for v0.9.1 can be found at [https://github.com/briis/unifiprotect/tree/v0.9.1](https://github.com/briis/unifiprotect/tree/v0.9.1)
```
Comment (AngellusMortis) [inline README.md:61]: Merged the updates. Also added a second about access docs for older versions.

Comment (briis) [reply README.md:61]: Very good. Are you done so I can merge?

Comment (AngellusMortis) [reply README.md:61]: Yep, go for it.

**Pull request merged**